charts/kubewarden-controller/templates/webhooks.yaml

```
@@ -1,9 +1,48 @@
+# generate certificates
+{{ $dnsName :=  printf "%s-webhook-service.%s.svc" (include "kubewarden-controller.fullname" .) .Release.Namespace }}
```

I prefer to use the FQDN:

```suggestion
{{ $dnsName := printf "%s-webhook-service.%s.svc.cluster.local" (include "kubewarden-controller.fullname" .) .Release.Namespace }}
```